docs/dev/Doctest.md

```diff
@@ -145,7 +145,7 @@ Doctest is relying on the console/command line to run code examples in documenta
     * https://github.com/crate/crate/blob/master/docs/general/dql/selects.rst
 
 
-Similar to CarateDB using it’s CLI “crash”, we can make use of our own [SQL-CLI](https://github.com/opendistro-for-elasticsearch/sql-cli)
+Similar to CarateDB using it’s CLI “crash”, we can make use of our own [SQL-CLI](https://github.com/opendistro-for-elasticsearch/sql/tree/master/sql-cli)
```

Comment:
Probably a typo `CrateDB`? @zhongnansu

Reply:
Yeah it should be CrateDB https://github.com/crate/crate, please fix it, thx. @joshuali925

Reply:
Fixed, thanks

Reply:
Thanks zhongnan and Josh!